app/assets/javascripts/modules/dir_toggle.js

```diff
@@ -0,0 +1,48 @@
+// Module adds behavior to Preview/Close preview for file objects
+
+(function( global ) {
+  var Module = (function() {
+
+    return {
+      init: function() {
+        document.querySelectorAll('.sul-embed-container').forEach(function(container){
```

Comment:
Please use `data-*` selectors for behavior rather than classes.  This will help differentiate behavior from style.

Reply:
This is taken from https://github.com/sul-dlss/sul-embed/blob/master/app/assets/javascripts/modules/file_preview.js#L8. I'm reluctant to introduce different approaches in the codebase.